.rhiza/tests/sync/test_rhiza_version.py

```
@@ -95,15 +38,24 @@ def test_rhiza_version_exported_in_makefile(self, logger):
 
-        proc = run_make(logger, ["print-RHIZA_VERSION"], dry_run=False)
+        # Clear RHIZA_VERSION from environment to test the default value
+        import os
+        import subprocess
+
+        env = os.environ.copy()
+        env.pop("RHIZA_VERSION", None)
+
+        cmd = ["/usr/bin/make", "-s", "print-RHIZA_VERSION"]
+        logger.info("Running command: %s", " ".join(cmd))
+        proc = subprocess.run(cmd, capture_output=True, text=True, env=env)
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)